timesketch/frontend-ng/src/App.vue

```diff
@@ -0,0 +1,97 @@
+<!--
+Copyright 2019 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<template>
+  <v-app id="app">
+    <v-app-bar app clipped-right flat :color="$vuetify.theme.dark ? '' : 'white'">
+      <v-img src="/dist/timesketch-color.png" max-height="30" max-width="30" contain></v-img>
+      <v-toolbar-title class="ml-3"> timesketch </v-toolbar-title>
```

Comment:
s/timesketch/Timesketch ?

Reply:
No, this is part of the logo, and it is lowercase (same as today) :)